scripts/cf_subshell_scoped_login.sh

```diff
@@ -32,7 +32,7 @@ case $TARGET in
 esac
 
 TMPDIR=${TMPDIR:-/tmp}
```

Comment:
Actually, should this now be:
```suggestion
TMPDIR=${TMPDIR:-/tmp/}
```

Reply:
Do you mean in addition to my change, or instead of?

Reply:
Without this extra change, aren't you running `mktemp -d "/tmpcf_home.XXXXXX"`?